SYCL/InvokeSimd/clang_run_error/ESIMD_to_SPMD.cpp

```diff
@@ -0,0 +1,103 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Check that full compilation works:
+// RUN: not %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out 2>&1 | FileCheck %s
+//
+// Tests invoke_simd support in the compiler/headers
+
+/* Test case specification: Test and report errors if invoked ESIMD function
+ * calls SPMD function.
+ *
+ * This test case is based on the assumption that ESIMD functions can't call
+ * SPMD functions, as there is no mechanism for this described in ESIMD
+ * specification.
+ *
+ * In this test case, ESIMD_CALLEE_doVadd() calls SPMD_CALLEE_doVadd().
+ * Currently, the compilation fails, which is good.
+ */
+
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/ext/oneapi/experimental/invoke_simd.hpp>
+
+#include <functional>
+#include <iostream>
+#include <type_traits>
+
+using namespace sycl::ext::oneapi::experimental;
+namespace esimd = sycl::ext::intel::esimd;
+
+// 1024 / 16 = 64: There will be 64 iterations that process 16 elements each
+constexpr int Size = 1024;
+constexpr int VL = 16;
+// constexpr int VL = 8; // For some reason, this vector length is not
+// supported; 16 seems to be hardcoded somewere in Konst's code
```

Comment:
Hardcoding VL to 16 was removed quite a while ago, so please try with the new dpcpp builds - https://github.com/intel/llvm/releases/.

Also, please replace "Konst" with references to code and/or specs, as external readers don't have a clue about personalities.